modules/standard/HDFS.chpl

```diff
@@ -71,20 +375,20 @@ record hdfsChapelFileSystem {
 
 // --------- Connecting/disconnecting ---------
 
-// Connect to HDFS
+/* Connect to an HDFS filesystem on a single locale */
```

Comment:
Also, do we want to expose this function?  It feels like an implementation detail to me.